test/e2e/test-plugins/block-icons/index.js

```diff
@@ -0,0 +1,157 @@
+( function() {
```

Comment:
Is fine as is, but this plugin might be useful also to test other things. I see it applicable with very slight tweaks  to verify custom block categories hook.